jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java

```diff
@@ -79,6 +79,16 @@ public void testDifferentTableKey() {
     }
 
     @Test
+    public void testNoTableKey() {
```

Comment:
I find test method name confusing compared with the first statement in the test :)
However, it's difficult to suggest a name which wouldn't break the convention with testDifferentTableKey() test. Initially, I was thinking about the name along the lines of _testFilterSettingsWithoutKey_
Or maybe leaving the method name as it is and calling explicitly
```java
filterSettings.setKey(null);
```
for better clarity?